spec/models/artist_spec.rb

```diff
@@ -17,6 +19,8 @@
   it { should respond_to(:birth_date) }
```

Comment:
Honestly not real sure!  I modeled the item and artist model specs off of the image_spec.  I find I'm easy to convince that model tests aren't worth the effort, but then I end up with nothing but integration tests!